examples/layers/JSONLayers/FL_Buildings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "url"       : "http://wxs.ign.fr/72hpsel8j8nhb5qgdh07gcyp/geoportail/wfs?",
+    "protocol"  : "wfs",
+    "version"   : "2.0.0",
+    "axisOrder" : "inverted",
+    "id"        : "wfsBuilding",
+    "title"     : "BDTOPO_BDD_WLD_WGS84G:bati_remarquable,BDTOPO_BDD_WLD_WGS84G:bati_indifferencie,BDTOPO_BDD_WLD_WGS84G:bati_industriel",
+    "styleName" : "",
+    "extractStyle" : true,
+    "level"     : 16,
```

Comment:
bbox could be retrieved using  `GetCapabilities`.
But `level` is an (optional) information that the user can give to iTowns - I don't see how iTowns could make that choice on itself (I'm not talking about the specific case of the globe, but iTowns as a framework).
Maybe the API could support attaching to another layer at specific levels:
```js
scene.addLayer(layer, parentLayer, 16)
scene.addLayer(layer, parentLayer, [16, 17, 18])
```
But of course it only makes sense if parentLayer is based on levels...